crowbar_framework/config/routes.rb

```diff
@@ -123,5 +123,10 @@
   get "crowbar/:barclamp", action: "versions", controller: "barclamp"
   post "crowbar/:barclamp/1.0/:action/:id", controller: "barclamp"
 
+  get "installer", controller: "installer", action: "index", as: "installer"
+  get "installer/status", controller: "installer", action: "status", as: "installer_status"
+  post "installer/set_status", controller: "installer", action: "set_status", as: "installer_set_status"
```

Comment:
Metrics/LineLength: Line is too long. [104/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)